deployment/persistent_chains/testnet-2.0/testnet-2.0-launch.md

```diff
@@ -77,7 +78,7 @@ The following steps are expected to be done **before** the ceremony.
 
     3.1. Share VN's IP address
 
-    3.2. Share VN's `id` (`id` field in `dcld status` command output)
+    3.2. Share VN's `id` (`node_id` field in `dcld init` command output)
```

Comment:
Probably we can mention both ways (`dcld status` and `dcld init`). I think it makes sense to know how to do it with `dcld status` in case the output of `dcld init` is lost.

Reply:
done